### PR TITLE
Remove old legacy scripts references

### LINKED ISSUE #21235: [Updater] Could not find resource js/config.js to load
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Related Nextcloud forum thread:
https://help.nextcloud.com/t/jsresourceloader-error-could-not-find-resource-js-config-js-to-load/66979?u=michaing

### Steps to reproduce
1. Use the web based updater to update from any version between NC17 and NC19 to any newer version.
2. Check nextcloud.log

### Expected behaviour
No error log entries.

### Actual behaviour
```
{"reqId":"XXXX","level":3,"time":"2020-06-04T13:11:40+02:00","remoteAddr":"XXXX","user":"--","app":"jsresourceloader","method":"GET","url":"/nextcloud/","message":"Could not find resource js/config.js to load","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4115.5 Safari/537.36 OPR/70.0.3707.0 (Edition developer)","version":"19.0.0.9","id":"XXXX"}
```
But this only shows up a **single time** as very first log entry when using the web-based updater and I noticed **no actual issues**. Hence I mark this as **low priority**.

### Server configuration

**Operating system:** Debian Buster and Bullseye

**Web server:** Apache 2.4

**Database:** MariaDB 10.3

**PHP version:** PHP7.3 and PHP7.4

**Nextcloud version:** 17/18/19

**Updated from an older Nextcloud/ownCloud or fresh install:** Issue appears during update

**Where did you install Nextcloud from:** https://download.nextcloud.com/server/releases/

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.12.0
  - apporder: 0.10.0
  - calendar: 2.0.3
  - cloud_federation_api: 1.2.0
  - contacts: 3.3.0
  - contactsinteraction: 1.0.0
  - dav: 1.15.0
  - federatedfilesharing: 1.9.0
  - files: 1.14.0
  - files_rightclick: 0.16.0
  - files_trashbin: 1.9.0
  - files_versions: 1.12.0
  - impersonate: 1.6.0
  - logreader: 2.4.0
  - lookup_server_connector: 1.7.0
  - nextcloud_announcements: 1.8.0
  - notes: 3.4.0
  - notifications: 2.7.0
  - oauth2: 1.7.0
  - photos: 1.1.0
  - previewgenerator: 2.3.0
  - provisioning_api: 1.9.0
  - ransomware_protection: 1.7.0
  - settings: 1.1.0
  - spreed: 9.0.0
  - survey_client: 1.7.0
  - tasks: 0.13.1
  - twofactor_backupcodes: 1.8.0
  - updatenotification: 1.9.0
  - viewer: 1.3.0
  - workflowengine: 2.1.0
Disabled:
  - accessibility
  - admin_audit
  - comments
  - encryption
  - federation
  - files_external
  - files_pdfviewer
  - files_sharing
  - files_videoplayer
  - firstrunwizard
  - password_policy
  - privacy
  - recommendations
  - serverinfo
  - sharebymail
  - support
  - systemtags
  - text
  - theming
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "my.domain.org"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "19.0.0.12",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": true,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0
        },
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "logtimezone": "Europe\/Berlin",
        "trashbin_retention_obligation": "disabled",
        "versions_retention_obligation": "disabled",
        "skeletondirectory": "",
        "defaultapp": "apporder",
        "maintenance": false,
        "overwrite.cli.url": "https:\/\/micha.gnoedi.org\/nextcloud",
        "htaccess.RewriteBase": "\/nextcloud",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpsecure": "ssl",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "mysql.utf8mb4": true,
        "updater.release.channel": "beta",
        "tempdirectory": "\/mnt\/sda\/ncdata\/tmp",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "app_install_overwrite": [
            "apporder",
            "calendar",
            "impersonate",
            "ransomware_protection",
            "contacts",
            "spreed"
        ],
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Opera

**Operating system:** Windows 10

#### Web server error log
<details>
<summary>Web server error log</summary>

```
nothing
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
{"reqId":"XXXX","level":3,"time":"2020-06-04T13:11:40+02:00","remoteAddr":"XXXX","user":"--","app":"jsresourceloader","method":"GET","url":"/nextcloud/","message":"Could not find resource js/config.js to load","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4115.5 Safari/537.36 OPR/70.0.3707.0 (Edition developer)","version":"19.0.0.9","id":"XXXX"}
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
nothing
```
</details>

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix https://github.com/nextcloud/server/issues/21235

Bug present since stable16 apparently :shrug: 
Was removed by https://github.com/nextcloud/server/commit/8e9d259074619de0f137d749fc94dd4e580d2ec9 @ChristophWurst

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: core/js/core.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
 		"sharedialogshareelistview.js",
 		"public/publicpage.js",
 		"setupchecks.js",
-		"../search/js/search.js",
```
Comment (rullzer) [inline core/js/core.json:16]: Are you sure this one can go as well in the backports?

**Review by rullzer — CHANGES_REQUESTED** [review]

File: core/js/core.json (hunk 1 above)
Comment (skjnldsv) [reply core/js/core.json:16]: 20 yeah, I'll remove for the others indeed :)

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**